manifest.json

```diff
@@ -32,7 +32,7 @@
     },
     "settings": {
         "event_subscriptions": {
-            "request_url": "https://slackbot.glueopshosted.com/slack/events",
+            "request_url": "<your server's url>/slack/events",
```

Comment:
@NichArchA82 can we keep the request url as server url? I'm aware it's auth'd off but it'll probably help avoid some issues in the future.